patches/components-password_manager-core-browser-ui-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/password_manager/core/browser/ui/BUILD.gn b/components/password_manager/core/browser/ui/BUILD.gn
+index ba4c9464ef10e9f65c1615b39c93929ce6bb67a7..366378c4c091ca741589b1f1878902d4ee9d75b7 100644
+--- a/components/password_manager/core/browser/ui/BUILD.gn
++++ b/components/password_manager/core/browser/ui/BUILD.gn
+@@ -74,6 +74,7 @@ source_set("ui") {
+     ]
+     deps += [ "//third_party/zxcvbn-cpp" ]
+   }
++  import("//brave/components/password_manager/core/browser/ui/sources.gni") sources += brave_chrome_components_password_manager_core_browser_ui_sources deps += brave_chrome_components_password_manager_core_browser_ui_deps
```

Comment:
It's not clear to me why we're adding these files back into this target because the usages are behind `!BUILDFLAG(IS_ANDROID)` flags anyway.

Reply:
I guess it's for https://github.com/brave/brave-core/pull/29330/files#diff-df18fbe668e1694e62a9825d180bd13b74b1b0243a3d291a8929e1c2ebda1e51R17 which calls it directly, but in that case it can be in it's own target in Brave and not have to patch here. Internally the target can have a public_dep on `components/password_manager/core/browser/ui` for `!is_android` and otherwise include the sources directly